src/nft/components/collection/FilterButton.tsx

```diff
@@ -0,0 +1,70 @@
+import clsx from 'clsx'
+import { Box } from 'nft/components/Box'
+import * as styles from 'nft/components/collection/FilterButton.css'
+import { Row } from 'nft/components/Flex'
+import { FilterIcon } from 'nft/components/icons'
+import { useCollectionFilters } from 'nft/hooks'
+import { putCommas } from 'nft/utils/putCommas'
+
+export const FilterButton = ({
+  onClick,
+  isMobile,
+  isFiltersExpanded,
+  results,
+}: {
+  isMobile: boolean
+  isFiltersExpanded: boolean
+  results?: number
+  onClick: () => void
+}) => {
+  const { minPrice, maxPrice, minRarity, maxRarity, traits, markets, buyNow } = useCollectionFilters((state) => ({
+    minPrice: state.minPrice,
+    maxPrice: state.maxPrice,
+    minRarity: state.minRarity,
+    maxRarity: state.maxRarity,
+    traits: state.traits,
+    markets: state.markets,
+    buyNow: state.buyNow,
+  }))
+
+  const showFilterBadge =
+    minPrice || maxPrice || minRarity || maxRarity || traits.length || markets.length || buyNow ? true : false
```

Comment:
don't need the ? true : false

Reply:
actually this should be reversed

Reply:
then wrap it with a paren and do ! at the beginning imo